repo/packages/S/spark/6/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "packagingVersion": "2.0",
+    "postInstallNotes": "The Apache Spark DCOS Service has been successfully installed!\n\n\tDocumentation: https://docs.mesosphere.com/usage/managing-services/spark/\n\tIssues: https://docs.mesosphere.com/support/",
```

Comment:
Apache Spark DCOS Service is not an Apache project. 
So, `s/Apache Spark/Spark/g`